cmd/minishift/cmd/start.go

```diff
@@ -187,13 +189,21 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
+	automountHostfolders(host.Driver)
+
 	clusterUp(&config, ip)
 	if err := openshift.AddSudoersRoleForUser("developer"); err != nil {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
 }
 
+func automountHostfolders(driver drivers.Driver) {
+	if hostfolder.IsAutoMount() && hostfolder.IsHostfoldersDefined(false) {
```

Comment:
Not sure I am a big fan of passing flags in the method signature to control logging? Do we really need that and it your code it is even fixed to _false_. I think it makes the code hard to understand. I can imagine what _hostfolder.IsHostfoldersDefined()_ does by just reading the method name, but then I start wondering why you pass a _bool_. I agree that logging is a bit mess atm. There is issue #169 and I think this one is urgent.  We need a decent approach to this. Once we have a proper logging framework, I would expect that your message would depend on the log level requested, right? I really would like to avoid to introduce yet more ways of controlling logging. It will make #169 even harder. 

Once we address #169, we need to document the expected logging approach as part of the dev guide as well.